### ISSUE TITLE
Add DeepAlpha - RL + ML crypto trading bot

### ISSUE THREAD

**Issue by stefanoviana** [kengz/awesome-deep-rl#14]
Suggesting [DeepAlpha](https://github.com/stefanoviana/deepalpha) for **Applications > Finance**.

AI crypto trading with LightGBM + XGBoost ensemble and HMM regime detection. 72 features, 70.9% walk-forward accuracy. MIT licensed.
